### ISSUE TITLE
(Item request) Smaller power transmission poles.

### ISSUE THREAD

**Issue by pooing1234** [Electrical-Age/ElectricalAge#680]
I think they should be smaller poles like [this](http://c8.alamy.com/comp/A24JCY/telegraph-pole-carrying-power-and-telephone-lines-detail-uk-A24JCY.jpg)

The poles should be thinner, and can only carry voltages around 200v, and power up to 6KW.
These can be used to feed straight into houses, or smaller buildings.

They should be a version with a little light fixture as well.

**Comment by AMIDIBOSS** [comment]
Yea,  let's mount lamps on them! 
But i talked to @Baughn and he didn't like the idea

**Comment by pooing1234** [comment]
@AMIDIBOSS How come? I think this really would be beneficial.

**Comment by AMIDIBOSS** [comment]
Ya, tell him about this...

**Comment by Baughn** [comment]
It's just because I have my own plans. I have to keep game balance in mind as well.

But rest assured, you'll get something that'll do the job.

**Comment by pooing1234** [comment]
@Baughn Understood. Using the 200v cables is becoming a pain since they can't handle much power. I need something that can support way more than 2kw.

**Comment by AMIDIBOSS** [comment]
What's your plan?

**Comment by Baughn** [comment]
We do already have 800 and 3200V cables, I will point out.

**Comment by pooing1234** [comment]
@Baughn Then we need to keep placing DC to DC converters everywhere. Not good, especially for smaller houses.

**Comment by AMIDIBOSS** [comment]
@pooing1234 Same problem!
We'd like to make it realistic! We'd like to make it the same a outta window!

**Comment by pooing1234** [comment]
@AMIDIBOSS Yep. Or, you could implement a smaller DC to DC converter that can get mounted to a wall.
